extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java

```diff
@@ -0,0 +1,41 @@
+package io.quarkus.logging.sentry;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Level;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "log.sentry")
+public class SentryConfig {
```

Comment:
There should be an `enabled` flag.  If it is `false` then the run time recorder should return an empty `Optional`.